### PR TITLE
Xeno impreg now gets stopped by stasis

### LINKED ISSUE #47304: Stasis beds do not halt Xeno impregnation
Seems like an oversight, but apparently being in a stasis bed doesn't do anything to halt the progress of Xenos gestation despite being in complete stasis. Could be due to balance reasons but I found this interesting enough to report it.

### PR DESCRIPTION
**Pull request by nemvar**
## About The Pull Request

title. Xeno eggs where getting processed both by their owner and SSobj.
Fixes https://github.com/tgstation/tgstation/issues/47304

## Why It's Good For The Game

Bugs bad.

## Changelog
:cl:
fix: Xeno and changeling impregnation times now get stopped by stasis. Also fixes a bug where they were growing up almost twice as fast as intended.
/:cl:

### CODE REVIEW

**Comment by ATH1909** [comment]
> Also fixes a bug where they were growing up almost twice as fast as intended.

Could you halve the "intended" gestation time, then? I do appreciate this bug being fixed, but I feel like doubling the already long alien larva gestation times is going to negatively impact xeno balance/fun (especially for alien queens, who'll now have to wait around twice as long for a larva to emerge from the body of their first victim).

**Comment by PKPenguin321** [comment]
I kinda hate this since it makes xeno egg removal not very urgent

**Comment by nemvar** [comment]
It's now the time it took a few months ago. It's just that nobody noticed that it's actually half the time it used to be.

**Comment by shellspeed1** [comment]
Gonna throw my 2 cents in here, maybe instead of xeno's being frozen, we should make it so they don't gib when they hatch so doctors have a chance to save the patient.

File: code/game/objects/items/body_egg.dm [hunk 1]
```diff
@@ -9,10 +9,10 @@
 	..()
 	to_chat(finder, "<span class='warning'>You found an unknown alien organism in [owner]'s [zone]!</span>")
 
-/obj/item/organ/body_egg/New(loc)
+/obj/item/organ/body_egg/Initialize(loc)
```
Comment (AnturK) [inline code/game/objects/items/body_egg.dm]: Param is incorrect now.

**Review by AnturK — CHANGES_REQUESTED** [review]

**Pull request merged**